rskj-core/src/test/java/co/rsk/peg/PegUtilsGetTransactionTypeTest.java

```diff
@@ -1023,7 +1020,7 @@ void test_pegout_no_change_output(
     ) {
         // Arrange
         BtcTransaction btcTransaction = new BtcTransaction(btcMainnetParams);
-        btcTransaction.addInput(BitcoinTestUtils.createHash(1), FIRST_OUTPUT_INDEX, activeFederation.getRedeemScript());
+        btcTransaction.addInput(BitcoinTestUtils.createHash(1), FIRST_OUTPUT_INDEX, createBaseInputScriptThatSpendsFromTheFederation(activeFederation));
```

Comment:
```suggestion
        Script inputScript = createBaseInputScriptThatSpendsFromTheFederation(activeFederation);
        btcTransaction.addInput(BitcoinTestUtils.createHash(1), FIRST_OUTPUT_INDEX, inputScript);
```
I'd split it like this to make lines shorter

Reply:
Same in the other places where createBaseInputScriptThatSpendsFromTheFederation is called